### ISSUE TITLE
Setting for image upload directory (preferably with filename prefix)

### ISSUE THREAD

**Issue by Menn1s** [glushchenko/fsnotes#1026]
**Describe your feature request**
 It would be nice for there to be a setting that allows users to specify where images are placed when they are dragged and dropped into the md editor (drag and drop is awesome, so thanks for that!). 
So by default images are placed in assets/. If I specify `${filename}.assets/` then it will automatically create the directory with the file's name followed by .assets and store it there instead. Or `pictures/` to store in a pictures directory, etc.

**Additional context**
This is for MacOS.

**Comment by stefandesu** [comment]
I strongly agree. For Markdown files, the images are currently places in the folder `i` which feels unconventional. Choosing how images are saved would be great! How [Typora](https://typora.io) handles this could be a great example:

<img width="712" alt="Screen Shot 2021-01-22 at 19 38 30" src="https://user-images.githubusercontent.com/3275148/105531314-69f91900-5ce9-11eb-93df-cd5c39576109.png">

Here's the piece of code that deals with images/files, but I don't know enough about macOS development to be able to implement this myself:

https://github.com/glushchenko/fsnotes/blob/master/FSNotes/Helpers/ImagesProcessor.swift#L210-L254
